src/main/java/uk/ac/cam/cl/dtg/segue/api/AdminFacade.java

```diff
@@ -72,37 +73,15 @@
 
 import javax.annotation.Nullable;
 import javax.servlet.http.HttpServletRequest;
-import javax.ws.rs.BadRequestException;
-import javax.ws.rs.Consumes;
-import javax.ws.rs.DELETE;
-import javax.ws.rs.ForbiddenException;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
-import javax.ws.rs.core.Context;
-import javax.ws.rs.core.EntityTag;
-import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Request;
-import javax.ws.rs.core.Response;
+import javax.ws.rs.*;
+import javax.ws.rs.core.*;
 import javax.ws.rs.core.Response.Status;
-import java.io.BufferedReader;
-import java.io.File;
-import java.io.FileReader;
-import java.io.IOException;
-import java.io.StringWriter;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
+import java.io.*;
+import java.util.*;
```

Comment:
It's best -- if ugly and annoying, I admit -- to keep separate imports rather than wildcards. I know IntelliJ does this automatically. Check this out to disable the behaviour https://peterdev.pl/2019/10/18/why-you-should-avoid-star-imports-in-intellij-idea/

Reply:
I recommend changing IntelliJ to up the settings "Class count to use import with `*`" and "Names count to use static import with `*`" to 20, and add two exceptions to the "Packages to use Import with `*`":
 - `uk.ac.cam.cl.dtg.segue.api.Constants`
 - `uk.ac.cam.cl.dtg.isaac.api.Constants`

Reply:
(They're in Editor -> Code Style -> Java -> Imports)